src/main/java/data/JsonConverter.java

```diff
@@ -13,20 +13,24 @@
 // limitations under the License.
 package data;
 
-import java.io.File;
-import java.io.FileWriter;
-import java.io.BufferedWriter;
-import java.io.FileNotFoundException;
-import java.io.IOException;
+import java.io.*;
 import java.util.*;
 import org.json.JSONObject;
 
 /** A class that creates a config file from a Vendor object. */
 public class JsonConverter {
-  private static final String FILE_PATH_BASE = "data/";
+  private static final String FILE_PATH_BASE = "../../src/main/resources/";
```

Comment:
Since we don't want test code (including the test path) in our actual code, can you change this so that fileBasePath is a private final String that is initialized via a constructor?

You can optionally have one constructor with no argument that uses a static constant as a default, and another constructor with 1 argument that lets the user specify a base path.